charts/backingservices/charts/constellation/values.yaml

```diff
@@ -2,18 +2,18 @@
 enabled: true
 deployment:
   name: "constellation"
-# Cloud provider details. Accepted values are aws or gcp
+# Cloud provider details. Accepted values are aws
 cloudProvider: "aws"
 # For aws cloud provider enter your acm certificate ARN here.
 awsCertificateArn : arn:aws:acm:us-west-2:xxxxx:certificate/xxxxxxx
-domainName: helm-test.constellation.pega.io
+domainName: YOUR_CUSTOM_DOMAIN_NAME_HERE
 # Docker repos and tag for image
 docker:
     # If using a custom Docker registry, supply the credentials here to pull Docker images.
     registry:
-      url: "cirrus-docker.jfrog.io"
-      username: cameng2
-      password: d8538894e2
+      url: YOUR_DOCKER_REGISTRY_URL
+      username: YOUR_DOCKER_REGISTRY_USERNAME
+      password: YOUR_DOCKER_REGISTRY_PASSWORD
     # Docker image information for the Pega docker image, containing the application server.
     constellation:
       image: cirrus-docker.jfrog.io/constellation-appstatic-service/docker-image:1.0.8-20221228123724
```

Comment:
@anuds20 @npjohnson637 - is it wise to hard code the Docker image AND TAG in this readme? What if it changes? I think the right thing to do is reference the Docker image name in the readme in a manner that follows the rest of the service team's Docker image references in the other yaml charts>>
pega values.yaml>>
pega:
      image: "pegasystems/pega"
# Elasticsearch deployment settings.
pegasearch:
  image: "pegasystems/search"
installer:
  image: "YOUR_INSTALLER_IMAGE:TAG"
hazelcast:
  image: "YOUR_HAZELCAST_IMAGE:TAG"
  clusteringServiceImage: "YOUR_CLUSTERING_SERVICE_IMAGE:TAG"

Reply:
@taz-pega-work Made the changes according to your comments. Also added a link to downloading pega docker images as requested.